css/team.css

```diff
@@ -0,0 +1,41 @@
+.section__team--grey {
+  background-color: #f5f4fa;
+}
```

Comment:
.page__secton--grey or team--grey, otherwise create a parent block for .section and then .section--grey would be a modificator of the block `section`